### ISSUE TITLE
Consider to adapt/merge fntsample_ucd_comments

### ISSUE THREAD

**Issue by eugmes** [eugmes/fntsample#7]
This fork looks interesting and possibly can be merged/adapted: https://github.com/ppablo28/fntsample_ucd_comments

**Comment by jsbien** [comment]
Just a reminder...

**Comment by jsbien** [comment]
Just a reminder...

**Comment by eugmes** [comment]
Yes, I still remember about that, just wanted to cleanup code first, but that progressing very slowly.

**Comment by jsbien** [comment]
Would you be so kind to look at plane15.pdf file at https://github.com/psb1558/Junicode-New/discussions/49?
Would it be easy to patch the original program to provide for line breaking?
